kube/src/api/typed.rs

```diff
@@ -58,15 +58,16 @@ where
 /// PUSH/PUT/POST/GET abstractions
 impl<K> Api<K>
 where
-    K: Clone + DeserializeOwned + Meta,
+    K: Clone + DeserializeOwned + Serialize + Meta,
```

Comment:
FWIW you can tag the relevant methods with `where K: Serialize` instead, so that `get` and `list` are still available for types without `Serialize`.

Reply:
Thanks! Fixed!